hoot-core/src/main/cpp/hoot/core/ops/RemoveNodeOp.cpp

```diff
@@ -64,7 +66,10 @@ void RemoveNodeOp::_removeNode(OsmMapPtr& map, long nId)
   const set<long>& ways = n2w->getWaysByNode(nId);
   if (ways.size() > 0)
   {
-    throw HootException("Removing a node, but it is still part of one or more ways.");
+    if (_removeOnlyUnused)
+      return;
+    else
+      throw HootException("Removing a node, but it is still part of one or more ways.");
```

Comment:
`_removeOnlyUnused` flag only removes unused nodes (i.e. orphaned or nodes that aren't a part of a way any longer).  Setting that flag keeps an exception from being thrown in that case.